nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/configuration/HipchatNotifier.java

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
done on the new ones. should I replace for all the payara-nucleus-modules that I have as well?